src/Error/ExceptionRenderer.php

```diff
@@ -142,7 +142,7 @@ protected function _getController()
                 $class = App::className('Error', 'Controller', 'Controller');
             }
 
-            /* @var \Cake\Controller\Controller $controller */
+            /** @var string $class */
```

Comment:
Controller was correct. just wrong doc block comment type.

Reply:
No, it's `$class` which type needed to be specified since `App::clasName()` can return null too. psalm is too smart and understands type declaration for right hand side of expressions too.

Reply:
But how does it then know the result is a controller?
I think in this case it would be cleaner to handle null, check for it (=== null) and then bail with a meaningful exception.
That way this hack of string annotation is not necessary and the original annotation that actually is useful can be kept. What do you think?

Reply:
In your IDE can you follow ` $controller->startupProcess();`? Can you click on it?